gapic/src/main/com/google/gapid/perfetto/views/FrameEventsSelectionView.java

```diff
@@ -56,6 +59,34 @@ public FrameEventsSelectionView(Composite parent, State state, FrameEventsTrack.
     createLabel(main, "Duration:");
     createLabel(main, timeToString(slice.dur));
 
+    if (slice.dur > 0) {
+      // If the selected event is a displayed frame slice, show the frame stats
+      Composite stats = withLayoutData(createComposite(this, new GridLayout(2, false)),
+          withIndents(new GridData(SWT.LEFT, SWT.TOP, false, false), PANEL_INDENT, 0));
+      withLayoutData(createBoldLabel(stats, "Frame Stats:"),
+          withSpans(new GridData(), 2, 1));
+
+      slice.frameStats.forEach((k, v) -> {
+        withLayoutData(createBoldLabel(stats, k.toString()),
+            withSpans(new GridData(), 2, 1));
+
+        createLabel(stats, "Queue to Acquire: ");
+        createLabel(stats, timeToString(v.queueToAcquireTime));
+
+        createLabel(stats, "Acquire to Latch: ");
+        createLabel(stats, timeToString(v.acquireToLatchTime));
+
+        createLabel(stats, "Latch to Present: ");
+        createLabel(stats, timeToString(v.latchToPresentTime));
```

Comment:
Can we show frameNumbers as well here? I think we should show users a little hint why something got grouped together by color. 
From the code I see they are connected by frameNumbers and layerNames, but these things are not shown in UI.

Reply:
Done.